### PR TITLE
New features for pagination

### PR DESCRIPTION
**Pull request by jlamim**
**Description**
The new features comprise a set of methods that will allow the user to build a pagination structure following the traditional way, where "next" and "prev" advance or return to a page and not to a group of pagination links as is the default.

Added methods:

- `hasPreviousPage:` checks if there is a page before the current page
- `getPreviousPage:` retrieves the URI of the page prior to the current page
- `hasNextPage:` checks if there is a page after the current page
- `getNextPage:` retrieves the page's URI after the current page

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPdocs
- [x] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by lonnieezell** [comment]
Looks good. Thanks! 

It does need new tests added for the new methods and something added to the docs about this.

**Comment by jlamim** [comment]
@lonnieezell I'm already working on testing and updating the documentation. Soon I will give the commits.

File: system/Pager/PagerRenderer.php [hunk 1]
```diff
@@ -343,4 +343,78 @@ protected function updatePages(int $count = null)
...
+	 * Returns a URL to the "next" page.
+	 *
+	 * You MUST call hasNextPage() first, or this value may be invalid.
+	 *
+	 * @return string|null
+	 */
+	public function getNextPage()
+	{
+		if (!$this->hasNextPage()) {
+			return null;
```
Comment (samsonasik) [inline system/Pager/PagerRenderer.php:407]: this returns null needs test

File: system/Pager/PagerRenderer.php [hunk 2]
```diff
@@ -343,4 +343,78 @@ protected function updatePages(int $count = null)
...
+	 * Returns a URL to the "previous" page.
+	 *
+	 * You MUST call hasPreviousPage() first, or this value may be invalid.
+	 *
+	 * @return string|null
+	 */
+	public function getPreviousPage()
+	{
+		if (!$this->hasPreviousPage()) {
+			return null;
```
Comment (samsonasik) [inline system/Pager/PagerRenderer.php:369]: this return null need test

File: system/Pager/PagerRenderer.php (hunk 1 above)
Comment (jlamim) [reply system/Pager/PagerRenderer.php:407]: OK! I will write the tests and update.

File: system/Pager/PagerRenderer.php (hunk 2 above)
Comment (jlamim) [reply system/Pager/PagerRenderer.php:369]: OK! I will write the tests and update.

**Comment by lonnieezell** [comment]
Looks good @jlamim Thanks for tackling this!

**Pull request merged**